src/shared/port-provider.js

```diff
@@ -135,24 +135,14 @@ export class PortProvider {
 
   /**
    * @param {'frameConnected'} eventName
-   * @param {(source: 'guest', port: MessagePort) => void} handler - this handler
+   * @param {(source: 'guest'|'sidebar', port: MessagePort) => void} handler - this handler
    *   fires when a request for the host frame has been granted. Currently, only
    *   the 'guest-host' channel triggers this listener.
```

Comment:
The second sentence here ("Currently, only the guest-host...") is out of date. I suggest to just remove it, as the type signature makes it obvious what the source frames may be.

Reply:
👍🏼